builds/dumbo/views/newListing.html

```diff
@@ -26,7 +26,7 @@ <h1 ng-if="newListingFormData.type"> {{newListingFormData.type}} </h1>
 				<div ng-if="photos.length || uploading">
 					<div class="imageHolder">
 						<a class="photoUploadBox" ng-repeat="image in photos track by $index" target="_blank">
-							<div ng-style="{'background-image':'url(' + image.photo_url + ')'}" class="scaledImage photoUploadImage"></div>
+							<div ng-style="{'background-image':'url(' + (image.photo_url | thumbnail) + ')'}" class="scaledImage photoUploadImage"></div>
```

Comment:
There more I think about it this it actually kind of creates a race condition that we don't want. User uploads image > send to AWS > AWS lambda code we have resizes it > we show the picture. Before once we were done sending it to AWS, the picture was available. 

We don't want to have users upload full pictures and then promptly download full version of them. We might want to think about this a bit